ts/Core/Axis/Axis.ts

```diff
@@ -7780,6 +7775,12 @@ class Axis {
                             dashstyle: (options as any).dashStyle
                         });
                     }
+                } else {
+                    graphic.addClass( // #15222
+                        'highcharts-crosshair highcharts-crosshair-' +
+                        (categorized ? 'category ' : 'thin ') +
+                        (options as any).className
```

Comment:
When `className` does not exist, the class name looks strange: `class="highcharts-crosshair highcharts-crosshair-thin undefined"`.
I would add some condition here like `options.className ? options.className : ''` to prevent that.